osf/management/commands/sync_datacite_doi_metadata.py

```diff
@@ -10,31 +10,38 @@
 
 logger = logging.getLogger(__name__)
 
-def sync_datacite_doi_metadata(dry_run=True, retries=4):
+def retry(func, retries=4):
+    for i in range(retries):
+        try:
+            time.sleep(0.3)
```

Comment:
why the extra sleep here? This will double sleep on a retry.

Reply:
That was to rate limit __US__ from DDOSing ourselves, but probably unnecessary, removed.